swagger.yaml

```diff
@@ -4,8 +4,14 @@ info:
   description: Metatavu Home lambdas
   version: 1.0.0
 
+servers:
+  - url: https://api.severa.visma.com/rest-api
+
 security:
   - bearerAuth: []
+  - OAuth2:
+    - users:read
+  - ClientIdAuth: []
```

Comment:
This should not be necessary. Remember the specification is describing the lambda function we are calling, not whatever api's the lambda function is calling. We already have bearer auth set up for accessing the lambdas and this is sufficient.